packages/cli/src/Format.ts

```diff
@@ -76,7 +75,7 @@ Or specify a Prisma schema path
       datamodel: output,
     })
 
-    output = output.trimEnd() + os.EOL
+    output = output.trimEnd() + '\n'
 
```

Comment:
```suggestion
```

From @tomhoule 

Since it's already done by https://github.com/prisma/prisma-engines/blob/15c0579cf8f153388368beadf2185058b7135a17/libs/datamodel/core/src/reformat.rs#L71

We could remove the line instead

Reply:
And then `let output` can be turned into `const output`